From ee2dc14e9d7da76c701e114ce2dcf6139604adcb Mon Sep 17 00:00:00 2001
From: Jerry Jelinek <jerry.jelinek@joyent.com>
Date: Thu, 1 Dec 2016 19:30:42 +0000
Subject: [PATCH] OS-5820 nodiratime causes linux nfs mount fails with
 "Function not implemented" Reviewed by: Patrick Mooney
 <patrick.mooney@joyent.com> Approved by: Patrick Mooney
 <patrick.mooney@joyent.com>

---
 usr/src/lib/brand/lx/lx_brand/sys/lx_mount.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/usr/src/lib/brand/lx/lx_brand/sys/lx_mount.h b/usr/src/lib/brand/lx/lx_brand/sys/lx_mount.h
index 074a6017c6..f9b239150d 100644
--- a/usr/src/lib/brand/lx/lx_brand/sys/lx_mount.h
+++ b/usr/src/lib/brand/lx/lx_brand/sys/lx_mount.h
@@ -78,7 +78,7 @@ extern int lx_nfs_mount(char *, char *, char *, int, char *);
 				LX_MS_RDONLY | LX_MS_NOSUID | \
 				LX_MS_NODEV | LX_MS_NOEXEC | \
 				LX_MS_REMOUNT | LX_MS_NOATIME | \
-				LX_MS_BIND | LX_MS_SILENT | \
+				LX_MS_NODIRATIME | LX_MS_BIND | LX_MS_SILENT | \
 				LX_MS_STRICTATIME | LX_MS_NOSEC | \
 				LX_MS_BORN | LX_MS_ACTIVE | LX_MS_NOUSER)
 
-- 
2.21.0

